packages/cli/snapshots/integration/generators/relation.belongs-to.integration.snapshots.js

```diff
@@ -7,9 +7,7 @@
 
 'use strict';
 
-exports[
-  `lb4 relation checks generated source class repository  answers {"relationType":"belongsTo","sourceModel":"Order","destinationModel":"Customer"} generates Order repository file with different inputs 1`
-] = `
+exports[`lb4 relation checks generated source class repository  answers {"relationType":"belongsTo","sourceModel":"Order","destinationModel":"Customer"} generates Order repository file with different inputs 1`] = `
```

Comment:
Was the original version formatted by prettier perhaps?

Reply:
I think the modified one formatted by prettier. I also checked it with `npm run lint:fix`.